Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -77,6 +77,38 @@ final class UIViewControllerExtensionsTests: XCTestCase {
 		}
 		XCTAssertEqual(alertController.preferredAction, alertController.actions[preferredButtonIndex])
 	}
+    
+    func testAddChildViewController() {
+        let parentViewController = UIViewController()
+        let childViewController = UIViewController()
+
+        XCTAssert(parentViewController.childViewControllers.isEmpty == true)
+        XCTAssertNil(childViewController.parent)
+        
+        parentViewController.add(childViewController)
+        
+        XCTAssert(parentViewController.childViewControllers == [childViewController])
+        XCTAssertNotNil(childViewController.parent)
+        XCTAssertEqual(childViewController.parent, parentViewController)
+    }
+    
+    func testRemoveChildViewController() {
+        let parentViewController = UIViewController()
```

Comment:
Can you add a test case that covers parent == nil ?

Reply:
I added a test case that checks if everything stays the same if you call remove on a view controller that has no parent. Is that what you meant?